src/mongodb/schemas.ts

```diff
@@ -0,0 +1,58 @@
+import { prop, getModelForClass /*, ReturnModelType*/ } from "@typegoose/typegoose";
+
+export class User {
```

Comment:
Would be better to separate each Schema into a different file

Reply:
The files aren't that big at all, it would make more clutter for me than actual utility if I separate it right now.